infra/.env.example

```
@@ -10,3 +10,35 @@ REDIS_PASSWORD=changeme_redis_password
 MINIO_BUCKET=aqsha
+
+# ── Langfuse (observability) — WAJIB HANYA bila menjalankan profile `langfuse`:
+#   docker compose -f infra/compose.dev.yaml --profile langfuse up -d
+# UI di http://$BIND_HOST:3000 (login pakai LANGFUSE_INIT_USER_EMAIL/PASSWORD).
+# Blob event/media numpang MinIO app pada bucket di bawah (default: langfuse).
+# LANGFUSE_S3_BUCKET=langfuse
+# Kredensial MinIO khusus Langfuse (user non-root, di-provision minio-init, akses hanya bucket
+# langfuse). SECRET min 8 char. Jangan pakai MINIO_ROOT_*.
+LANGFUSE_S3_ACCESS_KEY=langfuse
+LANGFUSE_S3_SECRET_KEY=changeme_langfuse_s3
+LANGFUSE_POSTGRES_PASSWORD=changeme_langfuse_pg
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFUSE_POSTGRES_PASSWORD key should go before the LANGFUSE_S3_ACCESS_KEY key

(UnorderedKey)